ietf/meeting/urls.py

```diff
@@ -33,16 +33,8 @@
     url(r'^agenda/%(owner)s/%(schedule_name)s/delete$' % settings.URL_REGEXPS, views.delete_schedule),
     url(r'^agenda/%(owner)s/%(schedule_name)s/make_official$' % settings.URL_REGEXPS, views.make_schedule_official),
     url(r'^agenda/%(owner)s/%(schedule_name)s(\.(?P<ext>.html))?/?$' % settings.URL_REGEXPS, views.agenda),
-    url(r'^agenda/%(owner)s/%(schedule_name)s/week-view(?:.html)?/?$' % settings.URL_REGEXPS, views.week_view),
-    url(r'^agenda/%(owner)s/%(schedule_name)s/by-room/?$' % settings.URL_REGEXPS, views.agenda_by_room),
-    url(r'^agenda/%(owner)s/%(schedule_name)s/by-type/?$' % settings.URL_REGEXPS, views.agenda_by_type),
-    url(r'^agenda/%(owner)s/%(schedule_name)s/by-type/(?P<type>[a-z]+)$' % settings.URL_REGEXPS, views.agenda_by_type),
     url(r'^agenda/%(owner)s/%(schedule_name)s/new/$' % settings.URL_REGEXPS, views.new_meeting_schedule),
-    url(r'^agenda/by-room$', views.agenda_by_room),
-    url(r'^agenda/by-type$', views.agenda_by_type),
-    url(r'^agenda/by-type/(?P<type>[a-z]+)$', views.agenda_by_type),
     url(r'^agenda/by-type/(?P<type>[a-z]+)/ics$', views.agenda_by_type_ics),
-    url(r'^agenda/personalize', views.agenda_personalize),
```

Comment:
This one needs to redirect and not 404. It will be in many people's bookmarks by now. Preferably the redirect would take them to the neue-view already set up to let them do what they expect the link would let them do. (As opposed to them having to go click more buttons first).